components/camel-zipfile/src/main/java/org/apache/camel/dataformat/zipfile/ZipFileDataFormat.java

```diff
@@ -39,6 +39,7 @@
  */
 public class ZipFileDataFormat extends ServiceSupport implements DataFormat, DataFormatName {
     private boolean usingIterator;
+    private boolean supportIteratorForEmptyDirectory;
```

Comment:
Can we rename this to `allowEmptyDirectory' to make it shorter and use similar name we have done for other such option